web-server/v0.4/.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: node_js
```

Comment:
Does this file get used by the TravisCI we have in place?  Or do we need to merge this with the root `.travis.yml` file in order for this to be useful?

Reply:
This is not needed and will be removed under this PR.